offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationOverview.groovy

```diff
@@ -82,9 +82,9 @@ class QuotationOverview {
         final totalPriceWithSymbol = Currency.getFormatterWithSymbol().format(offer.getTotalCost())
         final taxesWithSymbol = Currency.getFormatterWithSymbol().format(offer.getTotalVat())
         final netPriceWithSymbol = Currency.getFormatterWithSymbol().format(offer.getNetCost())
+        final netWithOverheadsWithSymbol = Currency.getFormatterWithSymbol().format(offer.getNetCostsWithOverheads())
```

Comment:
Do we want to have that on the first page? I think it suffices to have it in the overview. If we fill up the first page it will restrict the text that can be placed there at one point.

Reply:
Furthermore, it also breaks the design, since the lines for signature and the total price are not on the same level anymore. I would argue, if not requested we will not provide the price on the first page but on the price summary page is sufficient. If the request comes in we can still add it.

Reply:
I do not know this. I added it everywhere the VAT was applied since I understood the issue this way.

Reply:
I would prefer to have it only on the page of the Kostenauflistung (QuotationDetails). The issue itself does not give so much information, therefore I would try to be conservative and add it to the summary only if necessary to prevent further issues that might occur otherwise.

Reply:
https://github.com/qbicsoftware/offer-manager-2-portlet/issues/758